src/rn2xx3.cpp

```diff
@@ -3,7 +3,7 @@
  *
  * @Author JP Meijers
  * @Author Nicolas Schteinschraber
- * @Date 18/12/2015
+ * @Date 28/12/2018
```

Comment:
I think this date is the date the file was created

Reply:
Makes sense to me. Will change.

Reply:
Seems like you changed the wrong file's date in your second commit.